docusaurus.config.js

```
@@ -3,9 +3,10 @@
 
 const path = require('path');
 
-const environment = process.env.OBJECTIV_ENVIRONMENT;
-const isStagingEnv = environment ? environment.startsWith('staging') : false;
-const isProductionEnv = environment ? environment.startsWith('prod') : false;
+const nodeEnv = process.env.NODE_ENV;
+const isProductionEnv = nodeEnv ? nodeEnv.startsWith('prod') : false;
+const objectivEnvironment = process.env.OBJECTIV_ENVIRONMENT;
+const isStagingEnv = objectivEnvironment ? (isProductionEnv && objectivEnvironment.startsWith('staging')) : false;
```

Very nice. I like how you check both environments in conjunction here.

We could also throw an error when staging is set on non-prod environments (unless we plan to have a dev-staging 🙄)

---

Good one. Think throwing the error is more of a nice-to-have though, as it doesn't interfere with any of the dev settings.

---

Sure. I was more aiming at notifying the dev of an impossible set of configuration options